checkov/ansible/checks/graph_checks/PanosPolicyNoSrcAnyDstAny.yaml

```diff
@@ -0,0 +1,40 @@
+metadata:
+  id: "CKV_PAN_7"
+  name: "Ensure security rules do not have 'source_ip' and 'destination_ip' both containing values of 'any'"
+  category: "NETWORKING"
+definition:
+  or:
```

Comment:
I think this should be a big `and:` or is it ok, when `source_ip` is `any`, but `destination_ip` is `10.0.0.0`? and vice versa.

Reply:
The latter.

Ok:
source = any, destination = !any (example, inbound hosting rule)
source = !any, destination = any (example, outbound internet access rule)

Not ok:
source = any, destination = any